README.md

```diff
@@ -36,7 +36,7 @@ Project Calico uses [etcd](https://github.com/coreos/etcd) to distribute informa
 The `calico-node` service is a worker that configures the network endpoints for containers, handles IP routing, and installs policy rules.  It runs in its own Docker container, and comprises
 + Felix, the Calico worker process
 + BIRD, the routing process
-+ a [Powerstrip](https://github.com/clusterhq/powerstrip) adapter to set up networking when Docker containers are created.
++ Experimental Docker [libnetwork](https://github.com/docker/libnetwork) to set up networking when Docker containers are created (this replaces the [Powerstrip](https://github.com/clusterhq/powerstrip) adapter).

```

Comment:
Isn't it Experimental Docker libnetwork plugin, though?  That's what is currently there (except for the word 'plugin').